src/FileIO.h

```diff
@@ -0,0 +1,28 @@
+#ifndef FILEIO_H
+#define FILEIO_H
+
+#pragma once
```

Comment:
That's actually does the same as the include guards above (more modern though).
I.e. you can remove the include guards.

Reply:
nice. very good to know!